docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,149 @@
+  
+- **Resuming VFR**:
+  - Recovering military aircraft must automatically revert to VFR at the following points:
+    - At the initial point when recovering via military stream landing pattern (initial and pitch).
+    - At Hi-Key.
+    - Following a touch and go, go-around, or visual overshoot when a local IFR aircraft has indicated an intention to join the circuit.
+
+## Runway Modes
+Single runway operations only.
+
+## Circuit Procedures
+
+Circuit operations occur within a `5NM` radius of ESL ARP, at the following altitudes:
+
+<table>
+  <thead>
+    <tr>
+      <th>Type</th>
+      <th>Normal Altitude</th>
+      <th colspan="2">Low Level Altitude</th>
+    </tr>
+    <tr>
+      <th></th>
+      <th></th>
+      <th>Day</th>
+      <th>Night</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Non Jet</td>
+      <td>A010</td>
+      <td>A004</td>
+      <td>Circling Minima</td>
+    </tr>
+    <tr>
+      <td>Jet</td>
+      <td>A015</td>
+      <td>A006</td>
+      <td>Circling Minima</td>
+    </tr>
+  </tbody>
+</table>
+
+ATC should avoid the use of `MAINTAIN/EXTEND DOWNWIND` except when operationally required for separation with IFR aircraft or sequencing crossing runway traffic.
+
+### Circuit Direction
+| Runway | Direction |
+| ------ | ----------|
+| 09     | Left      |
+| 22     | Left      |
+| 04     | Right     |
+| 27     | Right     |
+
+## Circuit Saturation
+ATC should declare the circuit saturated when aircraft exceeds the below table.
+When the circuit is saturated further circuits will be unavailable until numbers are within tolerance of the table.
+This number may be reduced by ATC due to complexity of operations and prevailing weather in order to ensure safety.
+
+| Time   | ESL Day | ESL Night | WSL Day | WSL Night |
+| ------ | ------- | --------- | ------- | --------- |
+| Day    | 6       | 5         | 5       | 5         |
+
+## Low Approach
+- By day, pilots of local aircraft may request a low approach (descent to 50FT AGL) when less than the applicable runway separation standard exists.
+- Pilots are responsible for ensuring that no collision risk exists and that there is suitable spacing to continue the approach and for the potential/subsequent go-around.
+
+## Standard Taxi Routes
+| Runway     | Outbound Route                   | Inbound Route                   |
+| ---------- | -------------------------------- | -------------------------------- |
+| Runway 27  | E2 or Eastern Check Bays         | D2                               |
+| Runway 09  | D2 or Western Check Bays         | E2                               |
+| Runway 22  | E2 or Eastern Check Bays         | J                                |
+| Runway 04  | J                                | D2                               |
```

Comment:
This is great stuff. Any chance we could get a diagram similar to the YSSY ADC page which shows these standard taxi routes in action?